hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -283,6 +285,11 @@
       .defaultValue("false")
       .withDocumentation("Allow duplicates with inserts while merging with existing records");
 
+  public static final ConfigProperty<String> SPILLABLE_DISK_MAP_TYPE = ConfigProperty
+      .key("hoodie.spillable.diskmap.type")
+      .defaultValue(ExternalSpillableMap.DiskMapType.BITCASK.name())
+      .withDocumentation("Enable usage of either Bitcask or RocksDb as disk map for External Spillable Map");
```

Comment:
can we use the enum type here directly. for instance 
.. usage of either " + ExternalSpillableMap.DiskMapType.BITCASK.name() + " or " .... 
so thats users who directly copy this value from config page will not have any issues when trying it out.